pkgs/by-name/to/tone/package.nix

```diff
@@ -1,33 +1,46 @@
-{ lib, fetchFromGitHub, buildDotnetModule, ffmpeg-full, dotnetCorePackages }:
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  ffmpeg-full,
+  stdenv,
+  versionCheckHook,
+}:
 
 buildDotnetModule rec {
   pname = "tone";
-  version = "0.1.5";
+  version = "0.2.3";
 
   src = fetchFromGitHub {
     owner = "sandreas";
-    repo = pname;
+    repo = "tone";
     rev = "v${version}";
-    sha256 = "sha256-HhXyOPoDtraT7ef0kpE7SCQbvGFLrTddzS6Kdu0LxW4=";
+    hash = "sha256-fe0fcX617vsRdu42FEan181Zh5Todt9FLtOuD+C7yqA=";
   };
 
   projectFile = "tone/tone.csproj";
   executables = [ "tone" ];
-  nugetDeps = ./nuget-deps.nix;
+  nugetDeps = ./deps.nix;
 
   dotnetInstallFlags = [
     "-p:PublishSingleFile=false"
   ];
 
-  dotnet-sdk = dotnetCorePackages.sdk_6_0;
```

Comment:
Even though currently .NET 8 is the default sdk, I think we should be explicit and keep `dotnet-sdk` and `dotnet-runtime` attrs. It could be useful for example when .NET 8 will become EOL and .NET 10 will be the default: instead of automatically upgrading to incompatible 10, this package would still work but will be marked insecure.

Reply:
👍

Reply:
Great to know! I'll keep that in mind for C# packages from now on.